backend/server/app/services/product_service.py

```diff
@@ -20,4 +20,29 @@ async def process_query(self, query: str) -> Dict:
 
     async def update_query(self, query: str, answers: List[str]) -> str:
         updated_query = await self.llm_service.update_issue_details([query] + answers)
-        return updated_query
\ No newline at end of file
+        return updated_query
+    
+     async def find_product(self, consumer_need: str) -> Product:
+        # First, check if the consumer need is detailed enough
+        is_detailed = await self.llm_service.is_prompt_detailed_enough(consumer_need)
```

Comment:
I think we should not perform any 'validation' at this development stage

Reply:
Soluation: temporarly replaced those lines with TODO

Reply:
Yes you are right.
This function is expired and we will not use it.